CliClient/locales/ko.po

```diff
@@ -474,8 +474,8 @@ msgid ""
 "list all the tags (use -l for long option)."
 msgstr ""
 "[note]로부터 [tag]를 지우거나 지정할 때, 아니면 [tag]와 관련된 노트의 목록을 "
-"확인할 때 <tag-command>로 \"add\", \"remove\" 또는 \"list\" 를 사용하실 수 있"
-"습니다. 또한 `tag list` 명령어로 모든 태그의 목록을 확인할 수 있습니다 (긴 태"
+"확인할 때 <tag-command>로 \"추가\", \"삭제\" 또는 \"목록\" 을 사용하실 수 있"
```

Comment:
There's no tag command `추가`. The tag command is `add`. The same is true for the other commands.

Reply:
Translation was done manually in vscode.
Can't we?
I turned on the poredit when I saw your mail, but the ko.po file was not supported because it was json file.

Reply:
Or can I fix that part you mentioned?

Reply:
> I turned on the poredit when I saw your mail, but the ko.po file was not supported because it was json file.

I have no idea what you are talking about. The file `CliClient/locales/ko.po` is not json.

> Translation was done manually in vscode.
Can't we?

Apparently not, because you broke the file. Look at the Travis CI error.

Reply:
sorry i mistaken soon i'll fix it

Reply:
> Or can I fix that part you mentioned?

I gave only an example. Several of the translations were actually commands and/or arguments. I'm currently on my way to an appointment, otherwise I could point them out.

Reply:
> ile was not supported because it was json file.

Ah, you probably opened the wrong file. The json files are generated from the `.po` files.
`.po` files are always text files.